src/components/Tables/Table.tsx

```diff
@@ -135,28 +249,175 @@ const Table: React.FC<Props> = (props) => {
         }
     };
 
-    if (!domLoaded) {
-        return <></>;
-    }
+    const filterKeys = props.filters ?? props.headers.map(([headerKey]) => headerKey);
 
     return (
-        <DataTable
-            columns={columns}
-            data={dataToFilteredRows(props.data, filterText, props.headers)}
-            keyField="rowId"
-            subHeader
-            subHeaderComponent={
-                <TableFilterBar
-                    filterText={filterText}
-                    onFilter={onFilter}
-                    handleClear={handleClear}
-                    headers={props.headers}
+        <Styling>
+            <NoSsr>
+                <DataTable
+                    // types are fine without the cast when not using styled components, not sure what's happening here
+                    columns={columns}
+                    data={dataToFilteredRows(data, filterText, props.headers)}
+                    keyField="rowId"
+                    fixedHeader
+                    subHeader
+                    subHeaderComponent={
+                        <TableFilterBar
+                            filterText={filterText}
+                            filterKeys={filterKeys}
+                            toggleableHeaders={props.toggleableHeaders}
+                            onFilter={onFilter}
+                            handleClear={handleClear}
+                            headers={props.headers}
+                            setShownHeaderKeys={setShownHeaderKeys}
+                            shownHeaderKeys={shownHeaderKeys}
+                        />
+                    }
+                    pagination={props.pagination ?? true}
+                    persistTableHead
                 />
-            }
-            pagination
-            persistTableHead
-        />
+            </NoSsr>
+        </Styling>
     );
 };
 
+const RowDiv = styled.div`
+    display: flex;
+    width: 100%;
+    align-items: center;
+    padding-left: 1rem;
+`;
+
+const Spacer = styled.div`
+    width: 2rem;
+`;
+
+const EditandReorderArrowDiv = styled.div`
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
+    width: 100%;
+    // this transform is necessary to make the buttons visually consistent with the rest of the table without redesigning the layout
+    transform: translateX(-0.8rem);
+`;
+
+const StyledIcon = styled(FontAwesomeIcon)`
```

Comment:
Should this use Icon component over FontAwesome directly?

Reply:
Our icon component adds unwanted colour and padding